Admin/Admin.php

```diff
@@ -1,326 +1,368 @@
 <?php
 
-namespace BlueBear\AdminBundle\Admin;
+namespace LAG\AdminBundle\Admin;
 
-use BlueBear\AdminBundle\Manager\GenericManager;
+use ArrayIterator;
+use LAG\AdminBundle\Admin\Behaviors\ActionTrait;
+use LAG\AdminBundle\Admin\Behaviors\AdminTrait;
+use LAG\AdminBundle\Admin\Configuration\AdminConfiguration;
 use BlueBear\BaseBundle\Behavior\StringUtilsTrait;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\ORM\EntityRepository;
 use Exception;
+use LAG\AdminBundle\Admin\Message\MessageHandler;
+use LAG\AdminBundle\Exception\AdminException;
 use Pagerfanta\Adapter\DoctrineORMAdapter;
 use Pagerfanta\Pagerfanta;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Traversable;
 
-class Admin
+class Admin implements AdminInterface
 {
-    use StringUtilsTrait, ActionTrait;
+    use StringUtilsTrait, ActionTrait, AdminTrait;
```

Comment:
You might use container as static class for these methods instead of a StringUtilsTrait.